### ISSUE TITLE
[Bug] Fix the theme button position on small devices

### ISSUE THREAD

**Issue by Rakesh9100** [Rakesh9100/CalcDiverse#1916]
### Description

The theme-changing button currently appears inside the hamburger menu at last on small devices. It should be placed beside the hamburger icon for better accessibility and visibility on small devices.

### Screenshots

![image](https://github.com/user-attachments/assets/25a09e39-319e-4c84-94ec-57813ff66494)

### Checklist

- [X] I have checked the existing issues.
- [X] I follow [Contributing Guidelines](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CONTRIBUTING_GUIDELINES.md) of this project.
- [X] I am a KWOC'24 contributor.

### Code of Conduct

- [X] I agree to follow this project's "[Code of Conduct](https://github.com/Rakesh9100/CalcDiverse/blob/main/.github/CODE_OF_CONDUCT.md)".

**Comment by SHRUTISINHA250714** [comment]
Hi, I would like to work on this issue. Could you please assign it to me?

**Comment by abhirajadhikary06** [comment]
I would like to take up this issue. But as @SHRUTISINHA250714 appealed first assign this to her. If she is unable to complete within a specific time then I would make a PR.

**Comment by Rakesh9100** [comment]
Go ahead @SHRUTISINHA250714 

Don't create a PR until you are assigned to any issue @abhirajadhikary06

**Comment by abhirajadhikary06** [comment]
Go ahead @SHRUTISINHA250714 

> Don't create a PR until you are assigned to any issue @abhirajadhikary06 


Yeah I know the rules.. I will ask to assign before making aPR 👍
